erdjs-snippets/lending-pool/lendingPoolInteractor.ts

```diff
@@ -0,0 +1,348 @@
+import path from "path";
+import { AddressValue, BigIntValue, BytesValue, CodeMetadata, IAddress, Interaction, ResultsParser, ReturnCode, SmartContract, SmartContractAbi, Struct, TokenIdentifierValue, TokenPayment, TransactionWatcher } from "@elrondnetwork/erdjs";
+import { INetworkProvider, ITestSession, ITestUser, loadAbiRegistry, loadCode } from "@elrondnetwork/erdjs-snippets";
+import { NetworkConfig } from "@elrondnetwork/erdjs-network-providers";
+
+const PathToWasm = path.resolve(__dirname, "..", "..", "lending_pool", "output", "lending-pool.wasm");
+const PathToAbi = path.resolve(__dirname, "..", "..", "lending_pool", "output", "lending-pool.abi.json");
+const PathToWasmDummyLiquidityPool = path.resolve(__dirname, "..", "..", "liquidity_pool", "output", "liquidity-pool.wasm");
+
+export async function createInteractor(session: ITestSession, contractAddress?: IAddress): Promise<LendingPoolInteractor> {
+    let registry = await loadAbiRegistry(PathToAbi);
+    let abi = new SmartContractAbi(registry, ["LendingPool"]);
+    let contract = new SmartContract({ address: contractAddress, abi: abi });
+    let networkProvider = session.networkProvider;
+    let networkConfig = session.getNetworkConfig();
+    let interactor = new LendingPoolInteractor(contract, networkProvider, networkConfig);
+    return interactor;
+}
+
+export class LendingPoolInteractor {
+    private readonly contract: SmartContract;
+    private readonly networkProvider: INetworkProvider;
+    private readonly networkConfig: NetworkConfig;
+    private readonly transactionWatcher: TransactionWatcher;
+    private readonly resultsParser: ResultsParser;
+
+    constructor(contract: SmartContract, networkProvider: INetworkProvider, networkConfig: NetworkConfig) {
+        this.contract = contract;
+        this.networkProvider = networkProvider;
+        this.networkConfig = networkConfig;
+        this.transactionWatcher = new TransactionWatcher(networkProvider);
+        this.resultsParser = new ResultsParser();
+    }
+
+    async deployDummyLiquidityPool(deployer: ITestUser, tokenIdentifier: string): Promise<{ address: IAddress, returnCode: ReturnCode }> {
+        // Load the bytecode from a file.
+        let code = await loadCode(PathToWasmDummyLiquidityPool);
+
+        // Prepare the deploy transaction.
+        let transaction = this.contract.deploy({
+            code: code,
+            codeMetadata: new CodeMetadata(),
+            initArguments: [
+                new TokenIdentifierValue(tokenIdentifier),
+                new BigIntValue(0),
+                new BigIntValue(40000000),
+                new BigIntValue(1000000000),
+                new BigIntValue(800000000),
```

Comment:
@andreibancioiu , I don't like magic numbers in the code. Is there a nice way of aggregating them into an import file? Do you have an example?